### DISCUSSION TITLE
Evaluate between epochs

### DISCUSSION THREAD

**Discussion by aldro61** [pytorch/torchtune#2210] [Q&A]
Hi! 

I'm new to torchtune and I'm wondering if there is a way to evaluate on some validation set at the end of each epoch. Is there a mechanism to specify this in recipe config files? Otherwise what would be the best practice?

Thank you!

**Comment by aldro61** [comment]
Solved by creating a custom recipe using these instructions: https://pytorch.org/torchtune/0.5/basics/custom_components.html#launching-custom-recipes. Please let me know if there is a better way to do this. Thanks!
